aldryn_apphook_reload/utils.py

```diff
@@ -91,6 +91,7 @@ def reload_urlconf(urlconf=None, new_revision=None):
         reload(sys.modules[urlconf])
     cms.appresolver.clear_app_resolvers()
     django.core.urlresolvers.clear_url_caches()
+    django.core.urlresolvers.get_app_patterns()
```

Comment:
After fixing this import path typo: `cms.appresolver.get_app_patterns()` it solves the problem.